pkg/utils.go

```diff
@@ -257,13 +274,13 @@ func (initp *InitParams) setIPAndPorts(ctx context.Context) (err error) {
 
 // Get tls, info port
 func getPorts(ctx context.Context, k8sClient client.Client, namespace,
-	podName string) (infoPort, tlsPort int32, err error) {
+	podName string) (infoPort, tlsPort, adminPort, tlsAdminPort int32, err error) {
```

Comment:
Let's keep the naming consistent
```suggestion
	podName string) (servicePort, serviceTLSPort, adminPort, adminTLSPort int32, err error) {
```